mk/spksrc.common.mk

```diff
@@ -123,11 +123,17 @@ endif
 # Always send PSTAT output to proper log file
 # independantly from active Makefile location
 ifeq ($(filter cross diyspk spk,$(shell basename $(dir $(abspath $(dir $$PWD))))),)
+CROSSENV_LOG = $(shell pwdx $$(ps -o ppid= $$(echo $$PPID)) | cut -f2 -d:)/crossenv-$(ARCH)-$(TCVERSION).log
```

Comment:
I expected to name the new log files
- `build-crossenv-*.log`
- `build-wheel-*.log`

_without the build- prefix, I guess some clean* targets need an update._

but this can be done later...

Reply:
I tried that first but the ordering wasn't optimal.  Using `build-<arch>-<tcversion>-crossenv.log` and `build-<arch>-<tcversion>-wheel.log` would make things orderred when listing log files.

I suggest either using proposal above or trying current for a little while and reconvene later?

Reply:
your proposal sounds good, but as I said: no priority (I am longing for the merge)...